src/app/pages/auth/landing/landing.page.html

```diff
@@ -1,23 +1,89 @@
+          <ion-input formControlName="email" type="email" placeholder="user id"></ion-input>
+        </fieldset>
+      </ion-item>
+      <div style="padding: 20px;"></div>
+      <ion-item lines="none" class="ion-item">
+        <fieldset class="input-item">
+          <legend>Password</legend> 
+          <ion-input formControlName="password" type="password" placeholder="enter password"></ion-input>
+        </fieldset>
+      </ion-item>
+      <p class="signup-text">I'm a new user&nbsp;<a (click)="change('signup')">Sign Up?</a></p>
+      <ion-button class="login-button" (click)="login()" expand="block">Login</ion-button>
+      <p class="forget-text"><a (click)="change('forget')">Forgot Password?</a></p>
+      <!-- <ion-button class="google-button" expand="block" color="light"><ion-icon name="logo-google"></ion-icon>&nbsp;&nbsp;Google Signin</ion-button> -->
+    </form>
   </div>
-  <section class="section-config">
-    <div class="d-flex flex-justify-center">
-      <ion-button class="mx-0 onboard-button" expand="block" shape="round" size="large" (click)="onSignup()">
-        <ion-icon class="button-icon" name="mail-outline"></ion-icon>{{"SIGN_UP"}}
-      </ion-button>
-    </div>
-  </section>
-  <section class="section-config or-section">
-    <p><span>OR</span></p>
-  </section>
-  <section class="section-config">
-    <p class="text-config">{{"ALREADY_HAVE_ACCOUNT"}}</p>
-    <div class="d-flex flex-justify-center">
-      <ion-button class="mx-0 onboard-button" expand="block" shape="round" size="large" (click)="onLogin()">
-        <ion-icon class="button-icon" name="mail-outline"></ion-icon>{{"LOGIN"}}
-      </ion-button>
-    </div>
-  </section>
-</ion-content>
\ No newline at end of file
+
+  <!-- Sign up page -->
+  <div *ngIf="screen == 'signup'">
+    <h1 class="heading">🚀 Create Account<br><br></h1>
+    <p class="para">Sign up to get started!</p>
+    <form class="form" [formGroup]="formData">
+      <ion-item lines="none" class="ion-item">
+        <fieldset  class="input-item">
+          <legend>Full Name</legend> 
+          <ion-input formControlName="name" type="text" placeholder="user name"></ion-input>
+        </fieldset>
+      </ion-item>
+      <ion-item lines="none" class="ion-item">
+        <fieldset  class="input-item">
+          <legend>Email Id</legend> 
+          <ion-input formControlName="email" type="email" placeholder="user id"></ion-input>
+        </fieldset>
+      </ion-item>
+      <ion-item lines="none" class="ion-item">
+        <fieldset class="input-item">
+          <legend>Password</legend> 
+          <ion-input formControlName="password" type="password" placeholder="enter password"></ion-input>
+        </fieldset>
+      </ion-item>
+      <ion-item lines="none" class="ion-item">
+        <fieldset class="input-item">
+          <legend>Confirm Password</legend> 
+          <ion-input formControlName="cpassword" type="password" placeholder="confirm password"></ion-input>
```

Comment:
placeholder text like Enter your confirm password something like this make it in proper way